tv_apps/smart-home/js/home.js

```diff
@@ -696,18 +853,29 @@
       this.clock.start(this.updateClock.bind(this));
     },
 
-    onEditFolder: function() {
-      this._cardPicker.show(this.cardScrollable.currentItem);
-    },
-
     get focusElem() {
       return this._focus;
     },
 
     get focusScrollable() {
       return this._focusScrollable;
+    },
+
+    get mode() {
+      return this.mainSection.dataset.mode;
+    },
+
+    set mode(newMode) {
+      if (this.mainSection.dataset.mode != newMode) {
```

Comment:
But why you need to copy this property to home.js? Is there any benefit to do this than just use this in `edit.js`?